src/sonic-yang-models/tests/yang_model_tests/tests_config/interface.json

```diff
@@ -209,5 +209,33 @@
                 ]
             }
         }
+    },
```

Comment:
Please try to cover one negative scenario (e.g.  "ipv6_use_link_local_only": "true") where yang validation should fail

Reply:
Done.